### PR TITLE
Add dimension order arguments for netcdf data object

### PR DESCRIPTION
**Pull request by erikpelgrim**
Pull request for meerpeil correction fix.

### CODE REVIEW

File: core/java/src/org/openda/exchange/dataobjects/NetcdfUtils.java [hunk 1]
```diff
@@ -28,14 +28,7 @@
...
 import ucar.ma2.*;
-import ucar.nc2.NetcdfFile;
-import ucar.nc2.NetcdfFileWriter;
-import ucar.nc2.Variable;
-import ucar.nc2.Dimension;
-import ucar.nc2.Attribute;
-import ucar.nc2.NCdumpW;
-
-
+import ucar.nc2.*;
```
Comment (nilsvanvelzen) [inline core/java/src/org/openda/exchange/dataobjects/NetcdfUtils.java:31]: Codacy found an issue: [Avoid unused imports such as 'ucar.nc2'](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5780915)

**Comment by nilsvanvelzen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- core/java/test/org/openda/exchange/dataobjects/NetcdfDataObjectTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5780915&bid=18926811)

**Review by aljavrieling — CHANGES_REQUESTED** [review]
PLease remove the unused import statement: we do not want to introduce new Codacy issues.

Given the circumstances that OpenDA arrays do not know anything about the grid dimensions they originate from, this option is needed to match column-based and row-based data. The bugfix is OK.

**Comment by nilsvanvelzen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- core/java/test/org/openda/exchange/dataobjects/NetcdfDataObjectTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5780915&bid=18926811)

**Comment by nilsvanvelzen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Clones added
============
- core/java/test/org/openda/exchange/dataobjects/NetcdfDataObjectTest.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5780915&bid=18926811)

**Pull request merged**